### ISSUE TITLE
Allow displaying multiple tracks of reads

### ISSUE THREAD

**Issue by adamnovak** [vgteam/sequenceTubeMap#192]
Right now, we can display a single GAM file of reads.

We want to be able to display multiple read sets, probably using a more general track system from #191 instead of a single universal "GAM file" selector.

We will need the different read sets to be visually distinguishable. We also might need to come up with a more compact way to display reads, in order to make it possible to usefully look at a graph with ~60x read coverage on every base.

**Comment by adamnovak** [comment]
We might be able to start on this before we get the real UI in #205. We could just hardcode "GAM File 1" and "GAM File 2" in the UI and start working on getting multiple GAM files' reads extracted on the backend and being able to distinguish them visually in the tube map.

**Comment by adamnovak** [comment]
So the way I would approach this is:

1. Set up a hacky UI to allow specifying two GAM files. In addition to the GAM file dropdown, add another one with a new field in the state somewhere to keep its value, and set it up to control a new track that is added if a GAM file is selected.
2. Make the server parse the GAM file tracks as a list instead of just having one `gamFile` variable. Each GAM file should end up passed to `vg chunk` with a separate `-a` option.
3. See if that works and ends up with all the GAM reads from the different files mixed together.

**Comment by adamnovak** [comment]
For getting GAMs for testing you can do things like:

```
vg view -aj in.gam | head -n10 | vg view -JGa - >10reads.gam
vg view -aj in.gam | head -n20 | tail -n10 | vg view -JGa - >10morereads.gam
```

And then use the `scripts/` scripts to sort/index them

**Comment by adamnovak** [comment]
This was fixed in #213
